[2022-03-14 11:54:04,335] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: tutorial.cat.cat_get_data scheduled__2022-03-09T18:13:06.710203+00:00 [queued]>
[2022-03-14 11:54:04,352] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: tutorial.cat.cat_get_data scheduled__2022-03-09T18:13:06.710203+00:00 [queued]>
[2022-03-14 11:54:04,353] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-03-14 11:54:04,354] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-03-14 11:54:04,356] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-03-14 11:54:04,375] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): cat.cat_get_data> on 2022-03-09 18:13:06.710203+00:00
[2022-03-14 11:54:04,382] {standard_task_runner.py:52} INFO - Started process 209 to run task
[2022-03-14 11:54:04,391] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'tutorial', 'cat.cat_get_data', 'scheduled__2022-03-09T18:13:06.710203+00:00', '--job-id', '705', '--raw', '--subdir', 'DAGS_FOLDER/test_dag.py', '--cfg-path', '/tmp/tmplan1ot3n', '--error-file', '/tmp/tmps2vxs7ht']
[2022-03-14 11:54:04,393] {standard_task_runner.py:77} INFO - Job 705: Subtask cat.cat_get_data
[2022-03-14 11:54:04,513] {logging_mixin.py:109} INFO - Running <TaskInstance: tutorial.cat.cat_get_data scheduled__2022-03-09T18:13:06.710203+00:00 [running]> on host ff48034dfdbf
[2022-03-14 11:54:04,767] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=***@example.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=tutorial
AIRFLOW_CTX_TASK_ID=cat.cat_get_data
AIRFLOW_CTX_EXECUTION_DATE=2022-03-09T18:13:06.710203+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-09T18:13:06.710203+00:00
[2022-03-14 11:55:04,787] {python.py:175} INFO - Done. Returned value was: None
[2022-03-14 11:55:04,822] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=tutorial, task_id=cat.cat_get_data, execution_date=20220309T181306, start_date=20220314T115404, end_date=20220314T115504
[2022-03-14 11:55:04,885] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-03-14 11:55:04,981] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
